Task1b/Task1b.cs

```diff
@@ -12,9 +12,246 @@ namespace cg_lab3.Task1b
 {
     public partial class Task1b : Form
     {
+        Image imageToFill;
+        string selectedFileName;
+        Graphics g;
+        Point from, start_fill, to;
```

Comment:
Неправильное именование переменной
```suggestion
        Point from, startFill, to;
```